main/overview.md

```diff
@@ -46,6 +38,20 @@ The system is made up of five main units. These can be configured to work for a
 
 <p><b>Earlier hardware</b></p>
 
+```{image} img/emontx4-min.png
+:width: 50px
+:align: left
+```
+
+**emonTx v4:** A 6x circuit energy monitoring node. Transmits data via an inbuilt 433MHz radio to an emonBase. It can also send data via a Wi-Fi extension board or directly via USB.<br><br>
```

Comment:
Could  replace emonTx V3.4 section by emonTx4 section instead of having both emonTx here